Cargo.toml

```diff
@@ -64,6 +65,7 @@ http-body-util = "0.1.2"
 hyper = "1.4"
 hyper-util = { version = "0.1.6", features = ["client-legacy", "http1"] }
 hyper-tls = { version = "0.6.0", optional = true }
+hyper-rustls = { version = "0.27.2", features = ["webpki-roots"], optional = true }
```

Comment:
It's interesting.  The `reqwest` crate uses a dedicated `rustls-tls-webpki-roots` feature to enable webpki-roots but it's now noted as
> rustls-tls: Enables TLS functionality provided by rustls. Equivalent to rustls-tls-webpki-roots.

introduced by https://github.com/seanmonstar/reqwest/pull/1058

I'm not sure this is a good default (instead of `rustls-native-certs`), let's think about it

Reply:
It seems to be easy to add both `rustls-tls-webpki-roots` and `rustls-tls-native-roots` and make `rustls-tls` as default to `ustls-tls-webpki-roots` in the same way as the `reqwest` crate does it.

Reply:
My preference for webpki roots is only because native roots _can panic_ with no way to catch it:

https://github.com/rustls/hyper-rustls/blob/ceb2f61cc46b70806139962a1e8652c377ed292a/src/config.rs#L55

(This has happened to me in another project, and is really annoying because if you don't control where it's running, your only option is webpki)

As well as the fact that native certs is fallible outside of that, and I didn't quite want to make `new()` fallible as well.

Reply:
Ok, it makes sense. Let's leave only `rustls-tls` with implicit `webpki-roots` as implemented now